src/RepoAuditor/Module.py

```diff
@@ -133,7 +133,19 @@ def EvaluateQuery(
                     status_info.num_does_not_apply += len(query.requirements)
 
                     status_func(*status_info.__dict__.values())
-                    return 0, []
+                    # Since query returned None, it means it was not valid.
+                    return 2, [

```

Comment:
Will a return value of 2 cause the process to fail (since it is non-zero)? If so, would this cause GitHub actions to fail.

Reply:
Within `ParallelSequentialProcessor`, the return code is simply thrown away, with only the `EvaluateInfo` recorded in a `dict`. For that reason, this shouldn't be an issue.

On that note though, there is a lot of complexity in terms of the parallel execution which we could probably lessen.